src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -513,17 +516,33 @@ class UnifiedShareModal extends React.Component<Props, State> {
                 </div>
             </div>
         );
+        const ftuxTooltipProps = {
+            className: 'usm-ftux-tooltip',
+            isShown: shouldRenderFTUXTooltip && showCalloutForUser,
+            position: 'middle-left',
+            showCloseButton: true,
+            text: ftuxTooltipText,
+            theme: 'callout',
+        };
+        const recommendedSharingTooltipProps = {
+            className: 'usm-recommendedsharing-tooltip',
```

Comment:
Can these class names be changed to SUIT style?

Reply:
I guess they are old ones, nvm

Reply:
Though I dont see usm-recommendedsharing-tooltip being defined?

Reply:
you're right, className is not being used. edited!